app/iht/views/ihtHelpers/save_continue_button.scala.html

```diff
@@ -15,5 +15,5 @@
 *@
 
 <div class="form-field form-field--submit">
-  <input class='button' tabindex="0" type='submit' value='@Messages("iht.continue")'>
+  <input id="continue" class='button' tabindex="0" type='submit' value='@Messages("iht.continue")'>
```

Comment:
This should be called "continue-button" for consistency with uses elsewhere. E.g. app/iht/views/ihtHelpers/continue_button.scala.html